.github/workflows/tests.yml

```diff
@@ -10,7 +10,25 @@ jobs:
 
     strategy:
       matrix:
-        python-version: ['3.8', '3.10']
+        python-version: ['3.10']
+
+    services:
+      redis:
+          image: redis:alpine
+          ports:
+              - "6379:6379"
+
+      mongo:
+          image: mongo:4
```

Comment:
I believe Mongo v4 is already EOL. Do we support Mongo v6 in `develop` (I know we do in async branch, just not sure about the PyMongo version we're using here)

Reply:
seems working with 6